index.html

```diff
@@ -1426,7 +1434,7 @@ <h2>Chunk types</h2>
           </li>
         </ol>
 
-        <p>The remaining 18 chunk types are termed ancillary chunk types, which encoders may generate and decoders may
+        <p>The remaining 20 chunk types are termed ancillary chunk types, which encoders may generate and decoders may
```

Comment:
I suggest removing this sentence altogether. Why is the absolute number of chunk types important?

Reply:
Agreed. Will remove.